docs/build/tooling/README.md

```diff
@@ -13,6 +13,7 @@ This includes tools for development, operating a node, and ease of use of a Cosm
 * [Confix](./02-confix.md)
 * [Hubl](./03-hubl.md)
 * [Rosetta](https://docs.cosmos.network/main/run-node/rosetta)
+* [Ignite](https://docs.ignite.com)
```

Comment:
We need to modify the header as this page is about `various tooling maintained by the SDK team`.
Maybe let's add a section called `Third Party Tools` and precise those are great tools but outside the SDK team ownership.